matverse-discovery/src/pose_topological.py

```
@@ -0,0 +1,59 @@
+
+    try:
+        delta_gap = float(delta_gap)
+        l_xi = float(l_xi)
+        v_f = float(v_f)
+        k_b = float(k_b)
+        T = float(T)
+    except Exception as exc:  # pragma: no cover - defensive
+        raise ValueError("Parámetros inválidos") from exc
+
+    if T <= 0.0:
+        raise ValueError("La temperatura debe ser positiva")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)